### ISSUE TITLE
Add "Back" button

### ISSUE THREAD

**Issue by MagicLike** [openandroidinstaller-dev/openandroidinstaller#105]
It would come in handy to have a kind of "Back" button to go to the last step.

**Comment by tsterbak** [comment]
That's true, I thought about it but wasn't sure. Some of the steps shouldn't be retryed or went back from. So for less experienced users it seemed more dangerous than helpful.
What do you think?

**Comment by MagicLike** [comment]
Maybe a toggle for "Advanced Options" at the beginning (or in a kind of settings menu in the future)? This could also toggle the advanced output by default...

**Comment by tsterbak** [comment]
That's true. A settings menu with different options for advances users would be nice.

**Comment by nicorikken** [comment]
Tried OpenAndroidInstaller and initially the boot to recovery step failed. Ideally I would have gone back a couple steps to flash the custom recovery and try again. With version 0.4.3-beta I had to close the application and start over. A simple back button would have been nice, but a more guided 'boot to recovery failed, back to flashing custom recovery' would have even been clearer.

**Comment by nicorikken** [comment]
Yesterday in a workshop we struggled flashing a phone. Some of it was user error: accidentally disabling the cable, and clicking the wrong option in the Google Pixel bootloader screen. When stuck, there is no action available. The participant was hesitant to close OpenAndroidInstaller because of the risk of bricking the phone.

Rather than providing a 'back' button which introduces complexity in handling edge-cases, it could also be an option to 'start over' button that asks if you're sure with a warning and then starts from the beginning.

**Comment by tsterbak** [comment]
I agree, the current state is less than ideal. It sounds like a good idea to at least allow people to start over. I just need to make sure to clear all adb and fastboot activity cleanly, otherwise issues might just pile up. 

I will look into it :)
